Gemfile.lock

```diff
@@ -1,104 +1,103 @@
 GEM
```

Comment:
we should not have Gemfile.lock file as part of GitHub repo as deployment process will generate this file fresh always. Should we add this as part of gitignore?